.travis.yml

```diff
@@ -59,7 +59,7 @@ env:
     - PGVERSION=9.0 PQTEST_BINARY_PARAMETERS=no
 
 script:
- - go test -v ./...
+ - result=$(gofmt -d -e $(find -name \*.go)); test -z "$result" || (echo "$result" && false) && go test -v ./...
```

Comment:
Rather than a variable and `test`, could pipe the output to `awk`:

```
gofmt -d -e $(find -name \*.go) | awk '{ print } END { exit NR == 0 ? 0 : 1 }' && go test -v ./...
```